crates/staked-relayers/src/lib.rs

```diff
@@ -145,6 +145,11 @@ decl_module! {
 
         fn deposit_event() = default;
 
+        fn on_runtime_upgrade() -> Weight {
+            Self::_on_runtime_upgrade().expect("runtime upgrade failed");
```

Comment:
Not sure if Substrate catches these panics, perhaps it is better to just log the error. What do you think?

Reply:
The problem is that if this fails, the state of the sla pallet is invalid, so I figured it's better to panic fast rather than to operate in conditions where the invariant we rely on don't hold. The only way this can fail, though, is if the math operation under/overflow. Which is not impossible, since we use FixedPoint numbers to hold the sum of a bunch of multiplications (I think we might have to refactor this at some point), but currently is unlikely, since the chain is operating fine just now